share/picongpu/examples/Bunch/include/picongpu/param/radiation.param

```diff
@@ -65,7 +65,8 @@ constexpr unsigned int N_omega = 2048; // number of frequencies
 
 namespace frequencies_from_list
 {
-//const char listLocation[] = "/scratch/s5960712/list001.freq";
+/** path to text file with frequencies */
+const char listLocation[] = "/path/to/frequency.list";
```

Comment:
Muste be `constexpr char listLocation[]` to fit the documentation above. If this make compile issues change it to `constexpr char const listLocation* = "path";`

Reply:
I think actually `constexpr const char *` is preferable over `constexpr char listLocation[]` due to more operations allowed with such a variable w/o requiring a definition.